pkgs/by-name/ne/nexttrace/package.nix

```diff
@@ -23,6 +23,7 @@ buildGoModule rec {
     "-s"
     "-w"
     "-X github.com/nxtrace/NTrace-core/config.Version=v${version}"
```

Comment:
Better to attach a comment that refers to https://github.com/nxtrace/NTrace-core/issues/247